src/yamux/yamux.lua

```diff
@@ -355,6 +437,29 @@ local function parse_yamux(subtree, pinfo, fields, yamux_header)
         set_protocol(pinfo)
     end
 
+    if is_yamux_stream_default_created() and not (typeValue == hdr_type.PING or typeValue == hdr_type.GO_AWAY) then
+        dprint("Default opened stread cannot be with type", typeValue)
```

Comment:
```suggestion
        dprint("Default opened stream cannot be with type", typeValue)
```
Did you mean stream?